packages/integrations/react/context.js

```diff
@@ -0,0 +1,21 @@
+const contexts = new WeakMap();
+
+export function getContext(result) {
```

Comment:
Can we call `result` with a better name? It's a result of what?

Reply:
renamed `result` to `rendererContextResult`